api/v1alpha1/cluster_types.go

```diff
@@ -28,94 +28,72 @@ import (
 // CrdbClusterSpec defines the desired state of a CockroachDB Cluster
 // that the operator maintains.
 // +k8s:openapi-gen=true
-// +operator-sdk:csv:customresourcedefinitions:displayName="CoachroachDB Operator",resources={{StatefulSet,v1,cockroach-operator},{Service,v1,cockroach-operator}}
 type CrdbClusterSpec struct {
-	// (Required) Number of nodes (pods) in the cluster
+	// Number of nodes (pods) in the cluster
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Number of nodes",xDescriptors="urn:alm:descriptor:com.tectonic.ui:podCount"
 	// +required
 	Nodes int32 `json:"nodes"`
-	// (Required) Container image information
-	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Pod Image"
+	// Container image information
 	// +optional
 	Image PodImage `json:"image"`
-	// (Optional) The database port (`--port` CLI parameter when starting the service)
+	// The database port (`--port` CLI parameter when starting the service)
 	// Default: 26257
-	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="GRPC Port that defaults to 26257", xDescriptors="urn:alm:descriptor:com.tectonic.ui:number"
 	// +optional
 	GRPCPort *int32 `json:"grpcPort,omitempty"`
-	// (Optional)  The web UI port (`--http-port` CLI parameter when starting the service)
```

Comment:
yes... the generator that makes the field optional in CRD is **// +optional**. In CRD, the field description is build concatenating the **Required** tag and the comment for the fields marked  **// +required**. 
For the  fields that are optional will add only the comment in the CRD as field description. I removed this because the specDescriptor that are shown on OpenShift UI are also taken from comments. I will add them back for the fields that are  not in the OpenShift UI.

Reply:
Would it be possible to be consistent on our inclusion or exclusion of `(Optional)/(Required)` in field comments?

Given all the other magical annotations here, its very difficult to tell if those are magical incantations or not.